docker/templates/new-wagtail-app/{{ cookiecutter.app_name }}/{{ cookiecutter.module_name }}/static/js/hello-react.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
```

Comment:
Since this site could use both vanilla JavaScript (or jQuery) and React, what do you think of putting React code into `static/js/components/`.

Reply:
Sure, I can add an empty `components/` folder.